Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -1200,6 +1200,59 @@ def get_indices_statistics_command(args, proxies):
     )
     return result
 
+def kibana_list_cases_command(args, proxies):
+    kibana_url = demisto.params().get("kibana_url", "").rstrip("/")
+    api_key = demisto.params().get("kibana_api_key")
+    if not kibana_url or not api_key:
+        return_error("Kibana URL and API key must be configured in integration settings")
+    headers = {
+        "Authorization": f"ApiKey {api_key}",
+        "kbn-xsrf": "true",
+        "Content-Type": "application/json"
+    }
+    params = {}
+    if args.get("status"):
+        params["status"] = args["status"]
+    if args.get("owner"):
+        params["owner"] = args["owner"]
+    url = f"{kibana_url}/api/cases"
+    res = requests.get(url, headers=headers, params=params, verify=INSECURE, proxies=proxies)
```

Comment:
The direct use of `requests.get` is discouraged. It's recommended to implement a `BaseClient` class to centralize HTTP requests and consistently handle settings like timeouts, proxies, and certificate verification. Also, this request is missing a `timeout`, which could cause it to hang indefinitely.